test/basic-querying.test.js

```diff
@@ -201,9 +206,11 @@ describe('basic-querying', function() {
       User.find({order: 'order'}, function(err, users) {
```

Comment:
https://github.com/strongloop/loopback-connector-cassandra/blob/fix-array-conversion/test/cass.custom.test.js#L31

db.define('CASS_SORTABLE', {
      patBool: {type: Boolean, id: 2}, ====> composite partition key
      str: String,
      patStr: {type: String, id: true}, ====> composite partition key
      num: Number,
      patNum: {type: Number, id: 1}, ====> composite partition key
      yearMonth: {type: String, index: true}, ====> secondary key
      }, {
      cassandra: {
        clusteringKeys: ['str', 'num DESC'], ====> clustering keys for sorting
        },
      });